web/src/ui/settings/SettingsView.tsx

```diff
@@ -396,6 +396,11 @@ const SettingsView = ({ room }: SettingsViewProps) => {
 			err => window.alert(`Failed to request OpenID token: ${err}`),
 		)
 	}
+	const previousRoomId = roomMeta.creation_content?.predecessor?.room_id
```

Comment:
Go style variable name casing is superior to basic camel case, so it should be used in JS too (`Id` -> `ID`)